buzzmobile/process/bearing/bearing.py

```diff
@@ -43,8 +43,8 @@ def bearing(fix):
     currently no last_fix
     """
 
-    last_fix = globalVars['last_fix']
-    med_filter = globalVars['med_filter']
+    last_fix = globals['last_fix']
```

Comment:
nit: you can also use `globals['last_fix']` inline in the if statement then you can remove this line.